server/src/main/java/org/opensearch/index/mapper/MapperBuilderProperties.java

```diff
@@ -29,6 +29,15 @@ default Optional<DimensionType> getSupportedDataCubeDimensionType() {
         return Optional.empty();
     }
 
+    /**
+     * Indicates whether the current dimension is unsigned long.
+     *
+     * @return true if the dimension is unsigned long, false otherwise.
+     */
+    default Boolean isUnsignedLong() {
```

Comment:
Yeah lets not do this , mapperBuilderProperties is implemented by every builder, we can't introduce another new method specific for star tree.

Reply:
Yes - removed.